spec/models/order_spec.rb

```diff
@@ -1,5 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Order, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "has a valid factory" do
```

Comment:
I don't think we need to test "has a valid factory". We're already testing that a user can be created with the feature test. This would be just testing that FactoryGirl is working for our tests.